client/my-sites/marketing/main.jsx

```diff
@@ -152,12 +157,12 @@ export const Sharing = ( {
 					</NavTabs>
 				</SectionNav>
 			) }
-			{ ! isVip && ! isJetpack && (
+			{ ! isVip && ! isJetpack && currentPlanSlug && (
```

Comment:
Do we need to add any of these `currentPlanSlug` checks?  It seems to me like it might be OK to skip them.

Reply:
I haven't seen any flash of content or negative consequences after removing it. It's probably because of the extra checks on the Upsell component.
`currentPlanSlug` removed.